pkg/api/vm/formatter.go

```diff
@@ -234,6 +238,13 @@ func isReady(vmi *kubevirtv1.VirtualMachineInstance) bool {
 	return false
 }
 
+func canMigrate(vmi *kubevirtv1.VirtualMachineInstance) bool {
+	if err := virtualmachineinstance.ValidateVMMigratable(vmi); err != nil {
```

Comment:
Would suggest to update `ValidateVMMigratable` to update all errors to a straightforward description

`non-live migratable due to running` is a bit difficult to understand

```
	if !vmi.IsRunning() {
		return fmt.Errorf("VM %s considered non-live migratable due to running", vmiNamespacedName)

-> VM %s is not live migratable as it is not running
	}

	// The VM is already in migrating state
	if vmi.Annotations[util.AnnotationMigrationUID] != "" {
		return fmt.Errorf("VM %s considered non-live migratable due to migration state", vmiNamespacedName)

-> VM %s is not live migratable as it already has a running migration instance.

... others are similar
	}
```

Reply:
Sounds good. I'll update PR later.

Reply:
@w13915984028  Done.